### ISSUE TITLE
Black Screen Error

### ISSUE THREAD

**Issue by TheCamoWaffle** [sp614x/optifine#2085]
I'm using the latest 1.8.9 optifine mod and I randomly get black screens while playing.

**Comment by ozals** [comment]
Send a f3 screenshot please

**Comment by TheCamoWaffle** [comment]
Of the black screens?

**Comment by TheCamoWaffle** [comment]
The game keeps running just the screen goes black.

**Comment by ozals** [comment]
can you get into the game at all? if so, press the f3 key and send a screenshot of that
if you press f2, minecraft will take a screenshot and it will appear in \.minecraft\screenshots

**Comment by TheCamoWaffle** [comment]
To get back in the game I have to shut down my computer and start it again

**Comment by ozals** [comment]
wait, so does the game shut down, or your entire computer?

**Comment by ozals** [comment]
that
1. Sounds like a problem with your computer rather than optifine
2. Seems like you put it in sleep mode or something for it to not fully shut down
are you on a laptop by any chance?

**Comment by TheCamoWaffle** [comment]
Yeah I am, it only happens when playing Minecraft and it didn't happen when optifine wasn't on as a mod.

**Comment by TheCamoWaffle** [comment]
Do you have any idea as to why it might be happening?

**Comment by TheCamoWaffle** [comment]
It seems similar to a problem with 1.9 optifine someone had on here
